StreamingAnalytics/LowLatencySample/Temperature-Epl/monitors/TemperatureEventMonitor.mon

```diff
@@ -0,0 +1,29 @@
+package lowlatency;
+using com.apama.aggregates.mean;
+/** If average temperature in 5 minutes is greater than 80 C, send a shutdown alert  */
+
+event Temperature {
+    float temperature;
```

Comment:
I suggest naming this "value" - I think it would make the later code more readable (temperature.value instead of temperature.temperature).

Reply:
Done